### PR TITLE
feat(ui): add more prominent header

### LINKED ISSUE #61: Add Header similar to openui5.org

### PR DESCRIPTION
**Pull request by petermuessig**

### CODE REVIEW

**Comment by petermuessig** [comment]
Hi @marianfoo, please take a look... Maybe you like it?! 👀

File: src/view/App.view.xml [hunk 1]
```diff
@@ -5,10 +5,10 @@
         <OverflowToolbar style="Clear">
           <ToolbarSpacer />
           <Button
-            class="sapUiTinyMargin"
+            class="sapUiTinyMargin openui5headerButton"
             core:require="{ sapMLib: 'sap/m/library' }"
             text="{i18n>app_view_addyourpackage}"
-            type="Emphasized"
+            type="Transparent"
```
Comment (marianfoo) [inline src/view/App.view.xml:11]:
```suggestion
            type="Success"
```
i would prefer this to be a action button, clearly visible.
So either this color with `Success` or another color that clearly stands out.

**Review by marianfoo — CHANGES_REQUESTED** [review]
i like it!
few things on mobile:
## Button is not visible
image
## Content in Context Menu not visible
image
## minimize header
When on mobile, would be great if the header get smaller, either in general or when scrolling down

**Comment by petermuessig** [comment]
I thought about making it more lightweight. Regarding mobile - I think it may be better to fallback to the standard design - at least for the moment. Maybe in future we can get one designer on this topic. 😄

File: src/index.html [hunk 2]
```diff
@@ -61,6 +61,77 @@
...
+			.openui5headerButton:hover * {
+				background-color: transparent !important;
+			}
+			.openui5headerButton:hover .sapMBtnInner {
+				border-color: white !important;
+			}
+			.openui5headerButton:hover .sapMBtnInner:before,
+			.openui5headerButton:focus .sapMBtnInner:before {
+				border: none !important;
+			}
```
Comment (marianfoo) [inline src/index.html]: you mean removing this?
why is it in index.html and not in style.css anyway?

Comment (petermuessig) [reply src/index.html]: The splash stuff needs to stay in the index.html to be set early but the rest can be moved into the style.css. I'll have to recheck the mobile case and avoid the usage of the CSS overrides for the mobile scenario.

Comment (marianfoo) [reply src/index.html]: yeah, the splash stuff makes sense to stay in index.html
i think you can override it on mobile, just use a different color, that should work

Comment (petermuessig) [reply src/index.html]: I disabled the custom header for now on mobile and moved the CSS into the style.css. So we should be good to go.

Comment (petermuessig) [reply src/index.html]: Ah, regarding the buttons, I personally prefer to keep them flat. But this is a matter of taste IMO.

Comment (marianfoo) [reply src/index.html]: i´m fine with the look on desktop for now

**Comment by marianfoo** [comment]
as i mentioned before i dont like a prominent header on a content page
it´s different on openui5.org because this is a landing page
nevertheless, it is a nice header and looks way better than before
i reduced the height of the header a bit, are you fine with that?

**Comment by petermuessig** [comment]
I made some minor tweak about the position of the header and the big header will only be displayed when the screen size is wider than 600px.

**Review by marianfoo — APPROVED** [review]
LGTM!

**Pull request merged**